ssb-gitconfig/ssb-gitconfig.py

```diff
@@ -171,6 +174,8 @@ def set_base_config(pl: Platform) -> str:
         options = []
         if pl.prod_zone and pl.windows and pl.citrix:
             options = ["-c", "http.sslVerify=False"]
+        elif pl.prod_zone and pl.mac and pl.citrix:
```

Comment:
Det du sier her at det skal være et Mac-image som kjører på Citrix i prod-sona, og det har vi vel ikke?

Det jeg gjetter på at du ønsker å legge inn er Mac i adm-sona (dvs. tilkoblet med vpn med tilgang servere i adm-sona)?  Skal sette opp tilsvarende for Windows etterhvert, men har ikke kommet så langt ennå.